docs/admin/admission-controllers.md

```diff
@@ -139,5 +139,5 @@ Yes.
 For Kubernetes 1.0, we strongly recommend running the following set of admission control plug-ins (order matters):
 
 ```shell
---admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
+--admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,PersistentVolumeLabel,ResoureQuota
```

Comment:
This document should have a section describing what PersistentVolumeLabel does similar to the other choices.